app/renderer/components/Inspector/Source.js

```diff
@@ -72,9 +71,11 @@ class Source extends Component {
       if (elemObj.children.length === 0) {return null;}
 
       return elemObj.children.map((el) => {
-        return <TreeNode title={this.getFormattedTag(el)} key={el.path}>
-          {recursive(el)}
-        </TreeNode>;
+        return {
```

Comment:
this could be changed to 
```
return elemObj.children.map((el) => ({
 title: this.getFormattedTag(el),
 key: el.path,
 children: recursive(el),
}));
```